### PR TITLE
Video processing

### PR DESCRIPTION
**Pull request by ederev**
fixed video processing (face detection+recognition)

### CODE REVIEW

File: main/face_processing/utils/normalize_image.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import cv2
+import numpy as np
+
+# Below is the output of calibrate.py script
+DIM = (1280, 720)
+K = np.array(
```
Comment (andreytlt13) [inline main/face_processing/utils/normalize_image.py]: can we collect output from calibrate with out constant in a code?

File: main/video_processing.py [hunk 2]
```diff
@@ -267,26 +241,28 @@ def face_detection(self, frame, orig_frame, person_box, person_im):
                     # face box in person_im coordinates
                     box = detections[0, 0, i, 3:7] * np.array([W, H, W, H])
                     (startX, startY, endX, endY) = box.astype('int')
-                    pad_y, pad_x = 0, 0
+                    pad_y, pad_x = 30, 30
```
Comment (andreytlt13) [inline main/video_processing.py]: maybe we can use relative parameters for that?

File: main/video_processing.py [hunk 3]
```diff
@@ -267,26 +241,28 @@ def face_detection(self, frame, orig_frame, person_box, person_im):
...
                     face_im = person_im[startY - pad_y: endY + pad_y, startX - pad_x: endX + pad_x]
-                    # cv2.imwrite('cropped_face.jpg', face_im)
+
+                    if save_img:
+                        cv2.imwrite('face_processing/tmp_faces/tmp_cropped_face_{}.jpg'.format(tr_indx), face_im)
```
Comment (andreytlt13) [inline main/video_processing.py]: lets move it config and after collect each paths we can reorganised  it

**Comment by ederev** [comment]
[Issue #19: Video processing](https://trello.com/c/ByoQgIRf/22-issue-19-video-processing)

**Comment by ederev** [comment]
[Issue #19: Video processing](https://trello.com/c/ByoQgIRf/22-issue-19-video-processing)

**Pull request merged**